exporter.go

```diff
@@ -266,6 +266,9 @@ func (b *Exporter) Listen(e <-chan Events) {
 				}
 			} else {
 				eventsUnmapped.Inc()
+				if *dropUnmapped == true {
```

Comment:
Uh, please don't rely on a global variable. This setting should be an attribute of the exporter itself and configured when initializing the exporter.